### PR TITLE
Scholarships

### PR DESCRIPTION
**Pull request by jjheaney**
Changed link label per Farrah request.

Updated Scholarship emails per Mike request.

Avoids known problems with Salesforce's test system.

### CODE REVIEW

**Comment by jjheaney** [comment]
I'm happy to review and merge this myself if Thad says it's alright, but wasn't certain whether we had a guideline of a second pair of eyes before merging to master.

**Review by tcdahlberg — APPROVED** [review]

**Pull request merged**
